components/provisioner/README.md

```diff
@@ -1,10 +1,44 @@
 # Provisioner
 
+The Runtime Provisioner is a Compass component responsible for provisioning, installing, and deprovisioning clusters with Kyma (Kyma Runtimes).
```

Comment:
how does `(Kyma Runtimes)` relate to this description? I would describe it somehow.